activemodel/lib/active_model/railtie.rb

```diff
@@ -7,8 +7,15 @@ module ActiveModel
   class Railtie < Rails::Railtie # :nodoc:
     config.eager_load_namespaces << ActiveModel
 
+    config.active_model = ActiveSupport::OrderedOptions.new
+    config.app_generators.orm :active_model
```

Comment:
Weird. Why did you need to add this? Active Model is not a ORM.

Reply:
Removed.
Looking the active_record and http://api.rubyonrails.org/classes/Rails/Railtie.html as examples, I though it was needed.